README.md

```diff
@@ -1 +1,32 @@
-# engine-deploy-local
\ No newline at end of file
+# thirdweb Engine Deploy Locally
+
+Let's deploy thirdweb Engine locally in easiest way possible. This repository is maintained by thirdweb community team.
+
+> [!WARNING]
+> Use this for demo / preview purposes only. This is not recommended for production use. If you wanna use thirdweb Engine in production, try [cloud-hosted version](https://thirdweb.com/dashboard/engine?requestCloudHosted) or deploy a [self-hosted version](https://support.thirdweb.com/infrastructure/eRgkLPBdL1WJJLzAbuWrPZ/how-to-deploy-your-self-hosted-thirdweb-engine-on-the-railway/d97FnFt8e926FqniTaYxfD).
+
+## Requirements
+
+- [Docker](https://docs.docker.com/get-docker/)
+- [NodeJS](https://nodejs.org/en/download/)
+- [thirdweb API secret key](https://thirdweb.com/dashboard/api-keys)
+- [Wallet address](https://portal.thirdweb.com/glossary/wallet)
```

Comment:
Posgtres Version >= 14, should be added too

Reply:
Updated!